api/app/api/v1/routes/sync.py

```
@@ -0,0 +1,429 @@
+                index_elements=["booth_id", "local_id"]
+            )
+            
+            result = await db.execute(stmt)
+            
+            # rowcount == 1 means inserted, == 0 means conflict (duplicate)
+            if result.rowcount == 1:
+                synced_count += 1
+            else:
+                duplicate_count += 1
+                
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)